projects/1.20/assets/forestry-community-edition/forestry/lang/zh_cn.json

```
@@ -0,0 +1,2360 @@
+  "item.forestry.bee_comb_cocoa": "可可蜜脾",
+  "item.forestry.bee_comb_darkened": "暗黑蜜脾",
+  "item.forestry.bee_comb_dripping": "滴露蜜脾",
+  "item.forestry.bee_comb_frozen": "冻结蜜脾",
+  "item.forestry.bee_comb_honey": "蜂蜜蜜脾",
+  "item.forestry.bee_comb_irradiated": "辐射蜜脾",
+  "item.forestry.bee_comb_mellow": "醇厚蜜脾",
+  "item.forestry.bee_comb_mossy": "苔藓蜜脾",
+  "item.forestry.bee_comb_mysterious": "神秘蜜脾",
+  "item.forestry.bee_comb_omega": "终结蜜脾",
+  "item.forestry.bee_comb_parched": "干燥蜜脾",
+  "item.forestry.bee_comb_powdery": "粉末蜜脾",
+  "item.forestry.bee_comb_reddened": "泛红蜜脾",
+  "item.forestry.bee_comb_silky": "丝滑蜜脾",
+  "item.forestry.bee_comb_simmering": "激化蜜脾",
+  "item.forestry.bee_comb_stringy": "纤维蜜脾",
+  "item.forestry.bee_comb_wheaten": "麦香蜜脾",
+  "item.forestry.bee_comb_kaolin": "瓷土蜜脾",
+  "item.forestry.bee_comb_sponge": "海绵蜜脾",
+  "item.forestry.bee_comb_sculken": "幽匿蜜脾",
+  "item.forestry.beeswax": "蜂蜡",
+  "item.forestry.biome_finder": "栖息地定位仪",
+  "item.forestry.bituminous_peat": "沥青质泥炭",
+  "item.forestry.broken_bronze_pickaxe": "损坏的镐",
+  "item.forestry.broken_bronze_shovel": "损坏的锹",
+  "item.forestry.bronze_pickaxe": "生存家之镐",
+  "item.forestry.bronze_shovel": "生存家之锹",
+  "item.forestry.can.empty": "罐",
+  "item.forestry.can.grammar": "%s罐",
+  "item.forestry.carton": "纸箱",
+  "item.forestry.circuit_board_basic": "小型电路板",
+  "item.forestry.circuit_board_enhanced": "中型电路板",
+  "item.forestry.circuit_board_intricate": "大型电路板",
+  "item.forestry.circuit_board_refined": "先进电路板",
+  "item.forestry.circuit_board.tooltip.singular": "可安装%s个电子管。",
+  "item.forestry.circuit_board.tooltip.plural": "可安装%s个电子管。",
+  "item.forestry.dissipation_charge": "驱云药弹",
+  "item.forestry.dissipation_charge.tooltip": "在造雨机中用于停雨。",
+  "item.forestry.ice_shard": "碎冰",
+  "item.forestry.pulsating_dust": "脉动之尘",
+  "item.forestry.pulsating_mesh": "脉动线网",
+  "item.forestry.scented_paneling": "飘香镶板",
+  "item.forestry.silk_wisp": "丝缕",
+  "item.forestry.woven_silk": "丝绸",
+  "item.forestry.crate": "板条箱",
+  "item.forestry.decaying_wheat": "腐烂小麦",
+  "item.forestry.fertilizer_bio": "堆肥",
+  "item.forestry.fertilizer_compound": "肥料",
+  "item.forestry.frame_impregnated": "浸渍框架",
+  "item.forestry.frame_proven": "标定框架",
+  "item.forestry.frame_untreated": "未处理框架",
+  "item.forestry.frame_creative": "创造框架",
+  "item.forestry.durability": "耐久度：%1$s",
+  "item.forestry.uses": "可使用次数：%1$s",
+  "item.forestry.bee.modifier.production": "产量：%.2fx",
+  "item.forestry.bee.modifier.genetic.decay": "遗传衰减：%.2fx",
+  "item.forestry.fruit_cherry": "樱桃",
+  "item.forestry.fruit_chestnut": "栗子",
+  "item.forestry.fruit_dates": "椰枣",
+  "item.forestry.fruit_lemon": "柠檬",
+  "item.forestry.fruit_papaya": "木瓜",
+  "item.forestry.fruit_plum": "李子",
+  "item.forestry.fruit_walnut": "核桃",
+  "item.forestry.gear_bronze": "青铜齿轮",
+  "item.forestry.gear_copper": "铜齿轮",
+  "item.forestry.gear_tin": "锡齿轮",
+  "item.forestry.hardened_machine": "强化部件",

```

```suggestion
  "item.forestry.hardened_machine": "硬化外壳",
```
我是激进派.jpg
其他的就不一一修改了